### PR TITLE
Add argument to download script

### LINKED ISSUE #1: add architecture selection as argument to download script [feature]

### PR DESCRIPTION
**Pull request by oregonpillow**
Closes #1 

Just a suggestion, i found it useful to just enter the exact command than having to answer the prompt.  I think the prompt is nice as a fall back though for when no argument given.

### CODE REVIEW

**Review by chuckn246 — CHANGES_REQUESTED** [review]
I like your idea and will approve with a few changes.

- Revert changes to the colored output functions - keep them one line please.
- Same for the `1 | 2)` entry in the case statement
- Capitalize the s in the `# script arg` comment
- Wrap the `$arch_select` tests in curly braces (`${arch_select}`) 

Other than those personal preferences, I think it looks good!

Thanks for contributing and fixing my whitespace and backslash issues!

**Comment by oregonpillow** [comment]
Ok i _think_  i've addressed your points. Let me know if something still isn't fixed.
I was about to additionally make changes to the mkiso but realised i should save those for another issue/MR...

**Comment by chuckn246** [comment]
> Ok i _think_ i've addressed your points. Let me know if something still isn't fixed. 

Checking now!

> I was about to additionally make changes to the mkiso but realised i should save those for another issue/MR...

Yes please create another request for those changes, I was just about to mention that :)

File: get_latest_iso
Comment (chuckn246) [inline get_latest_iso:1]: Looks good, thank you for fixing my requests

**Review by chuckn246 — APPROVED** [review]

**Pull request merged**